src/index.scss

```diff
@@ -22,65 +25,69 @@
 }
 
 .dv-ChartContainer {
-  // position: static;
+  // this is a little hacky, sorry
+  margin-bottom: -1rem;
+
+  & + .dv-ChartContainer {
+    padding-top: 0;
+    margin-bottom: 0;
+  }
 }
 
 .dv-Sidebar {
   position: absolute;
-  top: 2.85rem;
-  left: 0;
+  top: 0;
+  right: 0;
   z-index: 100;
   overflow: hidden;
-  width: 50px;
-  height: calc(100% - 4.85rem);
+  width: 0;
+  bottom: 5.5rem;
```

Comment:
I dunno I just thought it was more elegant to give a bottom value than to have to do a height with `calc`